code/common/finspace.q

```diff
@@ -0,0 +1,44 @@
+// Config for setting Finspace specific parameters
+\d .finspace
+
+enabled:@[value;`enabled;0b];                               //whether the application is finspace or on prem - set to false by default
+database:@[value;`database;"database"];                     //name of the finspace database applicable to a certain RDB cluster - Not used if on prem
+hdbclusters:@[value;`hdbclusters;enlist `cluster];          //list of clusters to be reloaded during the rdb end of day (and possibly other uses)
+
+/ Runs a .aws api until a certain status has been received
+checkStatus:{[apiCall;status;frequency;timeout]
+  res:value apiCall;
+  st:.z.t;
+  l:0;
+  while[(timeout>ti:.z.t-st) & not any (res`status) like/: status; 
+     if[frequency<=ti-l;
+            l:ti;
+            res:value apiCall; 
+            .lg.o[`checkStatus;"Status: ", (res`status), " waited: ", string(ti)];
+     ];
+   ];
+   .lg.o[`checkStatus;"Status: ",(res`status)];
+   :res; 
+ };
+
+// Creates a Finspace changeset during the RDB end of day process
+createChangeset:{[db]
+      .lg.o[`createChangeset;"creating changeset for database: ", db];
+      details:.aws.create_changeset[db;([]input_path:enlist getenv[`KDBSCRATCH];database_path:enlist "/";change_type:enlist "PUT")];
+      .lg.o[`createChangeset;("creating changset ",(details`id)," with initial status of ",(details`status))];
```

Comment:
Can we please avoid changing the casing e.g. createChangeset
also apiCall above
Also .lg.o second variable doesn't need ()